src/com/twitter/intellij/pants/service/project/FastpassRecommendationsCounter.java

```diff
@@ -18,7 +18,7 @@ public boolean tick(Duration buildLength, LocalDateTime time) {
       }
       Duration sinceLast = Duration.between(lastTick, time);
       lastTick = time;
-      if(sinceLast.toMinutes() < 10 &&
+      if(sinceLast.toMinutes() <= 60 &&
```

Comment:
Should we make this number configurable via system properties? That way we won't need to republish the plugin binary in order to tweak the settings.